database/sampleDataMethods.js

```diff
@@ -19,21 +26,30 @@ module.exports.type = type;
 /* =================== Page Number ===================== */
 
 const pageNum = () => {
-  return faker.random.number({'min': 250, 'max': 800});
+  return getRandomInt(350, 450);
 };
 module.exports.pageNum = pageNum;
 
 /* =================== Publisher ===================== */
 
 const publisher = () => {
-  return faker.company.companyName();
+  let arr = ['Swaniawski Inc',
+    'Lueilwitz and Sons',
+    'Baumbach Inc',
+    'Rutherford Group',
+    'Shields - Torphy']
+
+  const num = getRandomInt(0, 4);
+  return arr[num];
 };
 module.exports.publisher = publisher;
 
 /* =================== Publication Dates ================== */
 
 const dates = () => {
-  return `${faker.date.month()} ${faker.random.number({ 'min': 1, 'max': 30 })}, ${(Math.floor(Math.random() * (2019 - 1960)) + 1960)}`
+  let month = ['January', 'March', 'April', 'May', 'June', 'July', 'October', 'November', 'December']
```

Comment:
I would recommend `months` as the variable name to indicate that we're working with an array.